### PR TITLE
470 change the color of the mark as duplicate button

### LINKED ISSUE #470: Change the color of the "mark as duplicate" button
Currently due to contrast issues with the yellow/gold color, the text is black.  All other buttons have white text.  Find a different color that will allow the text in the button to be white.

### PR DESCRIPTION
**Pull request by vanessuniq**
New look:

Screen Shot 2022-10-07 at 10 21 18 AM

Fixes #470

### CODE REVIEW

**Comment by vanessuniq** [comment]
Preview PR: https://vulcan-470-change-the-c-hqrdbp.herokuapp.com/users/sign_in

**Review by aaronlippold — APPROVED** [review]
Looks good to me. I am good with those colors.

File: app/javascript/application.scss [hunk 1]
```diff
@@ -37,3 +37,19 @@ $mdi-font-path: "~@mdi/font/fonts/";
 .white-space-pre-wrap {
   white-space: pre-wrap;
 }
+
+// Button background custum variant
+.btn-pink {
```
Comment (Amndeep7) [inline app/javascript/application.scss:42]: Do we still need the pink and purple button classes if we're moving forward with the orange?

**Review by Amndeep7 — CHANGES_REQUESTED** [review]

File: app/javascript/application.scss (hunk 1 above)
Comment (vanessuniq) [reply app/javascript/application.scss:42]: They were added to experiment, so no longer needed. will remove. Thanks!

**Review by Amndeep7 — APPROVED** [review]

**Pull request merged**